Tests/SwiftDocCTests/Rendering/RenderContentCompilerTests.swift

```diff
@@ -181,19 +181,46 @@ class RenderContentCompilerTests: XCTestCase {
         XCTAssertEqual(result.count, 6)
         do {
             guard case let .paragraph(paragraph) = result[0] as? RenderBlockContent else {
-                XCTFail("RenderCotent result is not the expected RenderBlockContent.paragraph(Paragraph)")
+                XCTFail("RenderContent result is not the expected RenderBlockContent.paragraph(Paragraph)")
                 return
             }
             let text = RenderInlineContent.text("\n")
             XCTAssertEqual(paragraph.inlineContent[1], text)
         }
         do {
             guard case let .paragraph(paragraph) = result[1] as? RenderBlockContent else {
-                XCTFail("RenderCotent result is not the expected RenderBlockContent.paragraph(Paragraph)")
+                XCTFail("RenderContent result is not the expected RenderBlockContent.paragraph(Paragraph)")
                 return
             }
             let text = RenderInlineContent.text("\n")
             XCTAssertEqual(paragraph.inlineContent[1], text)
         }
     }
+    
+    func testThematicBreak() throws {
+        let (bundle, context) = try testBundleAndContext(named: "TestBundle")
```

Comment:
Like in the other test, is is possible to use an empty bundle here instead of building and discarding the content?